target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.c

```diff
@@ -264,13 +259,13 @@ struct rtcl_round_set {
 
 static const struct rtcl_round_set rtcl_round_set[RTCL_SOCCNT][CLK_COUNT] = {
 	{
-		RTCL_ROUND_SET(300000000, 625000000, 25000000),
-		RTCL_ROUND_SET(200000000, 375000000, 25000000),
-		RTCL_ROUND_SET(100000000, 200000000, 25000000)
+		RTCL_ROUND_SET(300000000,	625000000,	25000000),
+		RTCL_ROUND_SET(200000000,	375000000,	25000000),
+		RTCL_ROUND_SET(100000000,	200000000,	25000000)
 	}, {
-		RTCL_ROUND_SET(400000000, 850000000, 25000000),
-		RTCL_ROUND_SET(100000000, 400000000, 25000000),
-		RTCL_ROUND_SET(50000000, 200000000, 50000000)
+		RTCL_ROUND_SET(400000000,	850000000,	25000000),
+		RTCL_ROUND_SET(100000000,	400000000,	25000000),
+		RTCL_ROUND_SET(50000000,	200000000,	50000000)
```

Comment:
These cosmetic changes aren't really related to making the 25MHz clock internal to the driver.